cilium/network_policy.cc

```diff
@@ -1080,6 +1081,9 @@ NetworkPolicyMap::NetworkPolicyMap(Server::Configuration::FactoryContext& contex
       local_ip_str_(context_.localInfo().address()->ip()->addressAsString()),
       name_(fmt::format("cilium.policymap.{}.{}.", local_ip_str_, ++instance_id_)),
       scope_(context_.serverScope().createScope(name_)),
+      stats_{ALL_CILIUM_POLICY_STATS(POOL_COUNTER_PREFIX(*scope_, "policy."),
```

Comment:
can you list an example of how this looks like in the prometheus output?

i wonder whether it would make sense to have configurable stats_prefix like it's often the case for envoy metrics.

Reply:
I'd find it easier to check how these look in a sysdump (where I think we also dump all Envoy metrics?) and then document this, rather than try to trace the source code and figure out exactly how these come out :-)

Reply:
yeah, that's what i meant. Just gathering an example output from the Envoy metrics endpoint (yes we do this when fetching the sysdump)